conans/test/functional/command/install_test.py

```diff
@@ -18,6 +18,61 @@ def setUp(self):
         self.settings = ("-s os=Windows -s compiler='Visual Studio' -s compiler.version=12 "
                          "-s arch=x86 -s compiler.runtime=MD")
 
+    def install_reference_txt_test(self):
+        # Test to check the "conan install <path> <reference>" command argument
+        client = TestClient()
+        client.save({"conanfile.txt": ""})
+        client.run("info .")
+        self.assertIn("conanfile.txt", str(client.out).splitlines())
+        client.run("install . Pkg/0.1@myuser/testing")
```

Comment:
This is confusing for *conanfile.txt* users

Reply:
It is not necessary, you don't need to specify it. Furthermore, raising an error was one of the possibilities suggested above. @jgsogo wanted to keep it to be consistent for all kinds of conanfiles